docs/manual.md

```diff
@@ -981,6 +981,61 @@ If you are booting from U-Boot, use the following command to start the system im
 Note that the OpenSBI version from the CVA6 SDK at the time of writing has issues when
 booting. It is recommended to use the mainline OpenSBI.
 
+## Cheshire
+
+Support is available for [Cheshire](https://github.com/pulp-platform/cheshire). It is an SoC design based on the CVA6 core, implementing a 64-bit RISC-V CPU. The design supports the hypervisor extension but currently is not supported in seL4.
```

Comment:
```suggestion
Support is available for [Cheshire](https://github.com/pulp-platform/cheshire). It is an SoC design based on the CVA6 core, implementing a 64-bit RISC-V CPU.
```

I would probably just omit this bit since it's not that useful if there's no seL4/Microkit support anyways.